Cargo.toml

```diff
@@ -297,6 +297,7 @@ targets = [
   "aarch64-unknown-linux-gnu",
   "aarch64-unknown-linux-musl",
   "arm-unknown-linux-musleabihf",
+  "armv5te-unknown-linux-gnueabi",
```

Comment:
We should remove this. This would only be relevant if we were building uv for this platform.